### PR TITLE
[WIP] Explain text predictions of Keras classifiers 

### PR DESCRIPTION
**Pull request by lopuhin**
https://github.com/TeamHG-Memex/eli5/pull/325/ by @teabolt 

TODO:
- [x] make sure build is passing
- [ ] double-check if anything needs to be updated / adjusted

### CODE REVIEW

File: docs/source/tutorials/keras-text-classifiers.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.. _keras-text-classifiers-tutorial:
+
+.. note::
+
+    This tutorial is intended to be run in an IPython notebook.
+    It is also available as a notebook file here_.
+
+.. _here: https://github.com/TeamHG-Memex/eli5/blob/master/notebooks/keras-text-classifiers.ipynb
```
Comment (teabolt) [inline docs/source/tutorials/keras-text-classifiers.rst]: Wondering if this link should be updated to point to the new org: `https://github.com/eli5-org/eli5/blob/master/notebooks/keras-text-classifiers.ipynb`. I know that other links will still be pointing to the old repo, but if this PR is only getting merged here then the link will be broken

**Comment by teabolt** [comment]
Hey @lopuhin

I checked out the docs that were built by tox and didn't find any issues. Added one small comment above but it might not even be an issue.

I also re-ran the keras notebooks using the py35-extra and py36-extra environments and didn't run into any issues. So I think the PR is ready

File: docs/source/tutorials/keras-text-classifiers.rst (hunk 1 above)
Comment (lopuhin) [reply docs/source/tutorials/keras-text-classifiers.rst]: Aha good point - I wanted to avoid updating the links unless it's needed (as I hope we can get back to old repo), but in this case it's indeed better to update, thanks for the heads up 👍

**Comment by teabolt** [comment]
Hey @lopuhin. Let me know if there's anything blocking this PR that I could fix. Maybe we are sticking with https://github.com/TeamHG-Memex/eli5 for now?

**Comment by lopuhin** [comment]
hi @teabolt I also think it's ready, just need to double-check everything and merge. We're sticking to this repo for now.
